src/datasources/transaction-api/transaction-api.service.spec.ts

```diff
@@ -47,6 +47,7 @@ describe('TransactionApi', () => {
 
     defaultExpirationTimeInSeconds = faker.number.int();
     notFoundExpireTimeSeconds = faker.number.int();
+    const messagesCache = faker.datatype.boolean();
```

Comment:
This test class needs a review as it stands right now, it seems incomplete and is probably covered by other tests.